src/IResourceStore.ts

```diff
@@ -0,0 +1,61 @@
+import IPatch from './IPatch'
+import IRepresentation from './IRepresentation'
+import IResourceIdentifier from './IResourceIdentifier'
+
+// Based on Ruben Verborgh's https://github.com/RubenVerborgh/solid-server-ts/blob/master/src/ldp
```

Comment:
Appreciate the credit; perhaps we should pick a standard way of listing authors per file. Something like JSdoc for TypeScript.